tally_ho/apps/tally/tests/views/data/test_form_list_view.py

```diff
@@ -58,3 +63,25 @@ def test_forms_for_race(self):
         response = view(request, ballot=ballot.number, tally_id=tally.pk)
         self.assertContains(response, "Forms for Race %s" % ballot.number)
         self.assertNotContains(response, "New Form")
+
+    def test_form_list_view_filter_on_Search_params(self):
+        """
+        #369 - check that filter search params are passed on
+        to data view.
+        """
+        tally = issue_369_result_forms_data_setup(self.user)
+
+        view = views.FormListView.as_view()
+        request = self.factory.get(
+            f'/1/?{pending_at_state_query_param}=data_entry_1&{race_type_query_param}=presidential'
+        )
+        request.user = self.user
+        request.session = {}
+
+        response = view(request, tally_id=tally.pk)
+
+        self.assertEqual(
+            response.context_data['remote_url'],
+            f"/data/form-list-data/{tally.pk}/?"
+            "pending_at_form_state=data_entry_1&race_type=presidential")
+        self.assertListEqual(response.template_name, ['data/forms.html'])
```

Comment:
Should we also test that the dictionary `response` has forms pending `data_entry_1` state?